sqlmesh/dbt/manifest.py

```diff
@@ -353,29 +400,7 @@ def _convert_jinja_test_to_macro(test_jinja: str) -> str:
     return re.sub(ENDTEST_REGEX, "{% endmacro %}", macro)
 
 
-def _extra_dependencies(target: str) -> Dependencies:
-    # We sometimes observe that the manifest doesn't capture certain macros referenced in the model.
-    # This behavior has been observed with macros like dbt.current_timestamp() and dbt_utils.slugify().
-    # Here we apply our custom extractor in addition to referenced extracted from the manifest to mitigate this.
-    dependencies = Dependencies()
-    for call_name, node in extract_call_names(target):
-        if len(call_name) == 2 and call_name[0] in ("dbt", "dbt_utils"):
-            dependencies.macros.append(MacroReference(package=call_name[0], name=call_name[1]))
-        elif call_name[0] == "source":
-            source = ".".join(_jinja_call_arg_name(arg) for arg in node.args)
-            if source:
-                dependencies.sources.append(source)
-        elif call_name[0] == "ref":
-            ref = ".".join(_jinja_call_arg_name(arg) for arg in node.args)
-            if ref:
-                dependencies.refs.append(ref)
-
-    return dependencies
-
-
 def _jinja_call_arg_name(node: nodes.Node) -> str:
-    if isinstance(node, nodes.Name):
```

Comment:
why was this removed?

Reply:
This is the crux of the error that was reported. Since we are statically parsing, we can only resolve constants - `source("package", "source")`, not vars - `source(package_var, source_table_var)`.

Reply:
so what happens if a var is passed?

Reply:
We don't include it in our supplemental dependency detection. It was never included before. This is all new since yesterday. The majority of source and ref calls are defined with constants, so that's the best bang for our buck. In the future, if we find dbt is missing a lot more dependencies from the manifest, we can look into adding spend more time here.